stdlib/pervasives.mli

```diff
@@ -319,23 +326,21 @@ val lnot : int -> int
 
 external ( lsl ) : int -> int -> int = "%lslint"
 (** [n lsl m] shifts [n] to the left by [m] bits.
-   The result is unspecified if [m < 0] or [m >= bitsize],
-   where [bitsize] is [32] on a 32-bit platform and
-   [64] on a 64-bit platform.
-   Right-associative operator at precedence level 8/11. *)
+    The result is unspecified if [m < 0] or [m > int_size].
```

Comment:
Isn't [int_size] just an alias for [bit_size]?
(If so, the characterization of unspecified
results has changed)

Reply:
No, `bit_size = 32 / 64` and `int_size = 31 / 63`

Reply:
Of course, sorry for the noise.